game/world/triggers/abstracttrigger.cpp

```diff
@@ -12,8 +12,6 @@ using namespace Tempest;
 
 AbstractTrigger::AbstractTrigger(Vob* parent, World &world, const zenkit::VirtualObject& data, Flags flags)
   : Vob(parent,world,data,flags & (~Flags::Static)), callback(this), vobName(data.vob_name) {
-  if(!hasFlag(StartEnabled))
```

Comment:
`StartEnabled` seen to be obsolete (and wrongly added in first place) - can be removed